### DISCUSSION TITLE
Zephyr v3.2.0 release plan date

### DISCUSSION THREAD

**Discussion by enthunilu** [zephyrproject-rtos/zephyr#48347] [General] [answered]
Hello,

I could see there is a recent release [Zephyr v3.1.0](https://github.com/zephyrproject-rtos/zephyr/releases/tag/zephyr-v3.1.0) happen on 5th June.

And in current month July, there are some important peripheral features committed to repository. So, just wanted to know the rough timeline for next release.

Also, I could see features at this moment planned for v3.2 [https://github.com/zephyrproject-rtos/zephyr/projects/9](https://github.com/zephyrproject-rtos/zephyr/projects/9)

Thanks!!

**Comment by DerekSnell** [comment, marked as answer]
Hi @enthunilu ,

Zephyr Project posts the [Release milestone dates](https://github.com/zephyrproject-rtos/zephyr/wiki/Release-Management#release-milestone-dates), and it shows the schedule for v3.2.0.

Best regards

**Reply by enthunilu** [reply]
Thank you so much @DerekSnell for quick response.
